### PR TITLE
Change verbose flag behaviour to output info-level an greater logs

### LINKED ISSUE #3088: Change hardcoded verbose flag back to using flag from the user
This has changed from being user configurable to being hardcoded, we should go back to letting it be configured: https://github.com/equinor/ert/blob/45ddbc60c96432c11ec44eb82bacf091f8d17307/libres/lib/analysis/update.cpp#L383

### PR DESCRIPTION
**Pull request by ManInFez**
**Issue**
Resolves #3088 

**Approach**
_Short description of the approach_

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by ManInFez** [comment]
test ert please!

File: libres/lib/enkf/obs_data.cpp [hunk 1]
```diff
@@ -128,12 +131,11 @@ static void obs_block_free__(void *arg) {
...
 }
 
-void obs_block_deactivate(obs_block_type *obs_block, int iobs, bool verbose,
+void obs_block_deactivate(obs_block_type *obs_block, int iobs,
                           const char *msg) {
     if (obs_block->active_mode[iobs] == ACTIVE) {
-        if (verbose)
-            printf("Deactivating: %s(%d) : %s \n", obs_block->obs_key, iobs,
-                   msg);
+        logger->info("Deactivating: {}({}) : {} \n", obs_block->obs_key, iobs,
```
Comment (oyvindeide) [inline libres/lib/enkf/obs_data.cpp:137]: This is perhaps a `WARNING`? It could lead to failure if all obs are deactivated.

**Review by oyvindeide — APPROVED** [review]
LGTM! Just one minor comment about log level

**Comment by ManInFez** [comment]
test ert please!

**Comment by ManInFez** [comment]
test ert please!

File: tests/ert_tests/conftest.py [hunk 2]
```diff
@@ -1 +1,15 @@
 # This conftest still exists so that tests files can import ert_utils
+import logging
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def log_check():
+    logger = logging.getLogger()
+    logger.setLevel(logging.WARNING)
```
Comment (oyvindeide) [inline tests/ert_tests/conftest.py:9]: Isnt this a bit high? Should be set to `DEBUG` to catch if any tests are actually modifying the log level?

File: ert_shared/main.py [hunk 3]
```diff
@@ -477,6 +477,10 @@ def main():
     logger = logging.getLogger(__name__)
     if args.verbose:
         logger.setLevel("DEBUG")
```
Comment (oyvindeide) [inline ert_shared/main.py:479]: Remove this, it is already set to debug in the logging configuration.

**Review by oyvindeide — APPROVED** [review]
Minor comments, but still looks good, nice to get the imports sorted!

File: tests/ert_tests/conftest.py (hunk 2 above)
Comment (ManInFez) [reply tests/ert_tests/conftest.py:9]: Warning is the default in pytest it seems, so leaving it at that

**Pull request merged**